src/blocks/tweet.v1.yaml

```diff
@@ -0,0 +1,40 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Tweet
+type: object
+properties:
+    type:
+        type: string
+        enum:
+          - tweet
+    id:
+        title: Tweet ID
+        type: string
+    date:
+        $ref: ../misc/partial-date.v1.yaml
+    text:
+        title: Text
+        type: array
+        items:
+            $ref: paragraph.v1.yaml
+        minItems: 1
```

Comment:
Do we have paragraph blocks, or a single string of HTML?

Reply:
> Do we have paragraph blocks, or a single string of HTML?

Single string of html.